WARNING:root:Too few points to create valid contours
ClusterEmcee2_linear_mu_2step.py:270: RuntimeWarning: invalid value encountered in log10
  err_b = 22.5-2.5*np.log10((mu_r * (f_0 - nsig*np.sqrt(sig_sq)))+mu_r)
WARNING:root:Too few points to create valid contours
WARNING:root:Too few points to create valid contours
/usr/local/lib/python2.7/dist-packages/matplotlib/axes/_axes.py:545: UserWarning: No labelled objects found. Use label='...' kwarg on individual plots.
  warnings.warn("No labelled objects found. "
('700', '800', 'normal')
Running object 700
0.0
{'i': 80, 'r': 68, 'z': 133, 'g': 62}
{'i': 1.2106731, 'r': 1.1613793, 'z': 1.1834742, 'g': 0.8314157}
Variable??
3.312876390406598
Non-Variable source in g
Variable??
4.299107169482074
Non-Variable source in r
Variable??
4.4516604562559445
Non-Variable source in i
Variable??
4.837062649847438
Non-Variable source in z
Slope:       [0.5016382743331718, 1, 0.4872257888431659, 0.6378476285633197]
Slope Error: [0.1395842335380725, 0, 0.11077523143558558, 0.13800547811308606]
intercepts:  [0.006602747063356618, 0, -0.017527646464872593, 0.026817604039614862]
Int Error:   [0.01318375336187731, 0, 0.01077730571752534, 0.013424482880087194]
StDev:       [0.087441, 0.11388774, 0.12997207, 0.19928946]
('ROW:', 700, 'Tau:', '2.5926407410763446e-05', 'V:', '0.05056737085227972')
Running object 701
0.0
{'i': 53, 'r': 41, 'z': 56, 'g': 8}
{'i': 0.56737006, 'r': 0.41210437, 'z': 0.70813215, 'g': 0.16550861}
Variable??
0.7782807306151709
Non-Variable source in g
Variable??
0.660937679107148
Non-Variable source in r
Variable??
0.552630902643921
Non-Variable source in i
Variable??
1.1359422051807382
Non-Variable source in z
Slope:       [1, 1, 1, 0.37189400240063963]
Slope Error: [0.5016258279432048, 0, 0.1212692280911048, 0.16221976339581362]
intercepts:  [-0.0704630146139086, 0, -0.022897827613050672, -0.013570231456319304]
Int Error:   [0.03852528352866891, 0, 0.015021139975506505, 0.019729444527202843]
StDev:       [0.021649983, 0.05480689, 0.063539594, 0.10354315]
('ROW:', 701, 'Tau:', '0.005352762057192928', 'V:', '-0.8986596137717973')
Running object 702
0.0
{'i': 71, 'r': 52, 'z': 89, 'g': 10}
{'i': 0.6944773, 'r': 0.49327093, 'z': 0.84862787, 'g': 0.17224091}
Variable??
1.2472881300325098
Non-Variable source in g
Variable??
0.9532744565110493
Non-Variable source in r
Variable??
1.1729813666317497
Non-Variable source in i
Variable??
1.9121984951990714
Non-Variable source in z
Slope:       [1, 1, 1, 0.3990766976243978]
Slope Error: [0.44717252135594626, 0, 0.14756945300967664, 0.19005808485629566]
intercepts:  [0.01616464956451887, 0, -0.0033711852018640798, 0.024372340732775832]
Int Error:   [0.0585657127696436, 0, 0.0158372427600799, 0.020553717832867406]
StDev:       [0.024364581, 0.05438046, 0.076052986, 0.13013104]
('ROW:', 702, 'Tau:', '0.00048509827745295275', 'V:', '-0.8746439827896128')
Running object 703
0.0
{'i': 80, 'r': 75, 'z': 156, 'g': 49}
{'i': 1.6624644, 'r': 1.2830101, 'z': 1.813478, 'g': 0.60927874}
Variable??
5.457479501941376
Non-Variable source in g
Variable??
7.219728905865148
Variable Source!
Variable??
8.714973492164841
Variable Source!
Variable??
6.862143328131729
Variable Source!
Slope:       [0.8037085832564812, 1, 0.7338521927061936, 0.6624694957229444]
Slope Error: [0.17702566312700682, 0, 0.058685310179578766, 0.05749462273574858]
intercepts:  [0.01555744314682006, 0, -0.026869496058254165, -0.014650086088262146]
Int Error:   [0.023014829372780075, 0, 0.008353580814958574, 0.007914378236972124]
StDev:       [0.11568931, 0.18366502, 0.22646658, 0.26679912]
('ROW:', 703, 'Tau:', '0.00011364262036573508', 'V:', '-0.5780100796360479')
Running object 704
0.0
{'i': 35, 'r': 39, 'z': 12, 'g': 21}
{'i': 0.464974, 'r': 0.39990732, 'z': 0.55523276, 'g': 0.26591304}
Variable??
0.0716359690715412
Non-Variable source in g
Variable??
0.13119983132274188
Non-Variable source in r
Variable??
0.4885699880107124
Non-Variable source in i
Variable??
1.705966842713399
Non-Variable source in z
Slope:       [1, 1, 1, 1]
Slope Error: [0.2508821867146525, 0, 0.2304269880008535, 0.8942165438033616]
intercepts:  [0.017650328938415893, 0, 0.022585435670017563, 0.031725755679565676]
Int Error:   [0.022993772699552346, 0, 0.027665317304182792, 0.05557197667164957]
StDev:       [0.025797525, 0.04992011, 0.071689524, 0.07359006]
('ROW:', 704, 'Tau:', '3.9925208194557458', 'V:', '-2.8982158803542046')
Running object 705
0.0
{'i': 36, 'r': 3, 'z': 100}
{'i': 0.4463843, 'r': 0.20049721, 'z': 0.8283801, 'g': 0.0}
Variable??
0
Non-Variable source in g
Variable??
0.9104138585913288
Non-Variable source in r
Variable??
1.1858097637693088
Non-Variable source in i
Variable??
0.643876175582409
Non-Variable source in z
Slope:       [1, 1, 1, 1]
Slope Error: [0.0, 0, 0.0, 0.0]
intercepts:  [0, 0, 0, 0]
Int Error:   [0.0, 0, 0.0, 0.0]
StDev:       [0.0, 0.014475826, 0.08404412, 0.1361132]
('ROW:', 705, 'Tau:', '0.8036906604977455', 'V:', '-0.949176850913866')
Traceback (most recent call last):
  File "ClusterEmcee2_linear_mu_2step.py", line 509, in <module>
    perform_emcee_step2(time, flux_mod, err_mod, dMu_dict, scale_dict, color_sort_ones, ROW, mu)
  File "ClusterEmcee2_linear_mu_2step.py", line 339, in perform_emcee_step2
    sausageplot_step2(max_theta[0], time, flux, max_theta[1], 5, err**2, dMu_dict, scale_dict, color_sort_ones, ROW, fig)
  File "ClusterEmcee2_linear_mu_2step.py", line 178, in sausageplot_step2
    ax4.legend()
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/axes/_axes.py", line 564, in legend
    self.legend_ = mlegend.Legend(self, handles, labels, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/legend.py", line 386, in __init__
    self._init_legend_box(handles, labels, markerfirst)
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/legend.py", line 655, in _init_legend_box
    fontsize, handlebox))
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/legend_handler.py", line 119, in legend_artist
    fontsize, handlebox.get_transform())
  File "/usr/local/lib/python2.7/dist-packages/matplotlib/legend_handler.py", line 476, in create_artists
    self.update_prop(coll, barlinecols[0], legend)
IndexError: tuple index out of range
